src/components/RouteOption/DockedEbikeRouteOptionDetails.jsx

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+function formatTimestamp(isoString) {
```

Comment:
nitpick - could we put these two format functions under the component? they're essentially an implementation detail of `DockedEbikeRouteOptionDetails` - better to start with the component first IMO (tho this is a matter of taste - feel free to ignore)